exampleCourse/serverFilesCourse/templates/prompt_in_collapsed_section.mustache.html

```diff
@@ -0,0 +1,22 @@
+Here is some text at the top of the template.
```

Comment:
Is this testing an edge case? I'd err on the side of removing this.

Reply:
This is testing an edge case for the `check_tags` function, because the list returned by `lxml.html.fragments_fromstring` is of mixed type (and unfortunately the type annotations for that library don't give great guarantees for that function).